worlds/pokemon_emerald/docs/rom changes.md

```diff
@@ -1,7 +1,20 @@
+## New Behaviors
+
+- The union room receptionist on the second floor of pokemon centers was reworked for wonder trading via Archipelago
```

Comment:
There were some inconsistencies that I fixed in [46f7c86](https://github.com/ArchipelagoMW/Archipelago/pull/2918/commits/46f7c86c42d9c9bb8da81295766fe84e202f2fbf) and [548db3e](https://github.com/ArchipelagoMW/Archipelago/pull/2918/commits/548db3e7bf3f1269657fbc08157bf74f8ec61dbc).

On anything rendered on the website, I try to use the accented e. When talking about the franchise or when it's otherwise part of a proper noun, I try to capitalize it. But otherwise, the word is such a common noun within the world's documentation, variable names, and comments that capitalizing it every time feels extraneous to me, or even looks weird sometimes.